src/server/home/index.njk

```diff
@@ -1,42 +1,123 @@
-{% extends 'layouts/page.njk' %}
+{% extends 'layouts/page.njk' %} {% block content %}
```

Comment:
presumably we should put the content block on a new line? 

Did prettifier recommend this?  :thinking:

Reply:
yes it has been done automatically by prettier, not sure why, and not sure how to override these decisions prettier makes